lib/helpers.coffee

```diff
@@ -33,3 +33,53 @@ module.exports =
         return syntax[0]
 
     return existingSyntax
+
+  ###*
+   * Attempts to resolve the root directory/project directory of the currently open file/editor instance
+   * @param {string} filePath - The currently active editor file path
+   * @return {Object|null} The project directory instance or null if no project root is found
+   ###
+  getRootDir: (filePath) ->
+    rootDir = null
+    atom.project.rootDirectories.forEach((dir) ->
```

Comment:
This can all be replaced with:
```coffeescript
atom.project.relativizePath(filePath)[0]
```

It returns the project path as Atom sees it, or `null` if it isn't open in a project.